### ISSUE TITLE
missing file: 4-applied-web/blue_yellow_app/templates/mytemplate.pt

### ISSUE THREAD

**Issue by stevepiercy** [mikeckennedy/python-for-entrepreneurs-course-demos#6]
In the video [Mapping routes to controller actions demo](https://training.talkpython.fm/player/course/python-for-entrepreneurs-build-and-launch-your-online-business/lecture/50807), there is a file present, but it is missing in the repository. As a result it is hard to follow along. I understand it is deleted toward the end, but it would be nice to follow along.

`4-applied-web/blue_yellow_app/templates/mytemplate.pt`

**Comment by danshorstein** [comment]
I think you can find that file here:  https://github.com/mikeckennedy/python-for-entrepreneurs-course-demos/tree/master/2-web-starter/blue_yellow_app/blue_yellow_app/templates

**Comment by stevepiercy** [comment]
Yeah, eventually I just copied the whole thing over. See my other [comment about my experience](https://github.com/mikeckennedy/python-for-entrepreneurs-course-demos/issues/7#issuecomment-284540902).

**Comment by mikeckennedy** [comment]
Hi Steve,

I think what is happening here is a misinterpretation of what this folder in the github repo represents. Let me know what you think.

The folder entitled **4-applied-web** is the final state of the code after `chapter 4`. Not the starter code for this chapter. That starter code for `chapter 4` is `chapter 2`. The final code for `chapter 4` is what is in that part of GitHub.

As you can see in this video part way through this chapter (so before we get to the end which this is suppose to represent), we have renamed and reorganized the templates as follows:

![screen shot 2017-03-06 at 3 50 59 pm](https://cloud.githubusercontent.com/assets/2035561/23635462/e44cc6ca-0284-11e7-88a3-d60724ed5f2d.png)

Here's that video:

https://training.talkpython.fm/player/course/python-for-entrepreneurs-build-and-launch-your-online-business/lecture/50822

So it is entirely on purpose that `mytemplate.pt` is missing because we deleted it. That's why you found it in chapter 2.

Make sense?

**Comment by stevepiercy** [comment]
Yes, thanks. Eventually I understood its purpose, but see my [follow up comment about the experience, and suggestion for your consideration](https://github.com/mikeckennedy/python-for-entrepreneurs-course-demos/issues/7#issuecomment-284580057).
